site/en/docs/extensions/mv3/cross-origin-isolation/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Cross-origin isolation"
+date: 2021-08-03
+#updated: 
+description: Overview of cross-origin isolation for extensions
+---
+
+[Cross-origin isolation](https://web.dev/cross-origin-isolation-guide/) enables a web page to use
+powerful features such as SharedArrayBuffer. An extension can opt into cross-origin isolation by
+specifying the appropriate values for [cross_origin_embedder_policy][coep-refdoc] and
+[cross_origin_opener_policy][coop-refdoc] manifest keys. For example, a manifest like the following
+example will allow the extension to opt into cross-origin isolation for its origin.
```

Comment:
```suggestion
[cross_origin_opener_policy][coop-refdoc] manifest keys. For example, a manifest like the following
snippet will allow the extension to opt into cross-origin isolation for its origin.
```

"For example, … the following example" reads oddly. To read a little more naturally, we could replace the second example with a synonym.